Data/market-hours/market-hours-database.json

```diff
@@ -356,6 +356,7 @@
         "11/29/2019": "13:00:00",
         "11/27/2020": "13:00:00",
         "11/26/2021": "13:00:00",
+        "11/25/2022": "13:00:00",
         "12/24/2009": "13:00:00",
         "12/24/2012": "13:00:00",
```

Comment:
There is another `"7/4/2020"` around line 26673, should it also be fixed? There are a couple of other `"12/25/2022"` too

Reply:
Options dates are updated! I verified as many of the major entries dates as I could, some of the distant dates did not have calendars to verify, but nothing seemed out of the ordinary with alignment with previous years.

Reply:
Just checking: I see 10 matches of `12/25/2022`, here we are only changing 2, that's okay?

Reply:
Yes the other 8 entries are for markets that are open everyday not weekday markets. They all have Christmas closed on the actual day.